includes/TypesForLines/LineHinge.js

```diff
@@ -1,31 +1,160 @@
+// nonlinearity nefungují
+
+
+/**
+* Creates line hinge
+* @functiom
+* @constructor
+* @param	{Number}	no				Index of line hinge, can be undefined
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params			line hinge parameters, can be undefined
+* @return	{Object}	Created line hinge
+*/
+var createLineHinge = function (no,
+								comment,
+								params) {
+
+	if (no != undefined) {
+		var lineHinge = line_hinges.create(no);
+	}
+	else {
+	 var lineHinge = line_hinges.create();
+	}
+	set_comment_and_parameters(lineHinge, comment, params);
+
+	return lineHinge;
+};
+
+
+/**
+* Creates line hinge constant
+* @functiom
+* @param	{Boolean or Float}	hinge		hinge input (true, false, number(stiffness))
+* @return	{Object}	Created hinge constant
+*/
+function CreateHingeConstant(hinge) {
+  var hinge_constant = line_hinges["SPRING_CONSTANT_NO"];
+
+  if (hinge === true) {
+    hinge_constant = line_hinges["SPRING_CONSTANT_YES"];
+  }
+  return hinge_constant;
+}
+
+
+function CreateHinge(hinge) {
+  if (hinge === true || hinge === false) {
+    return CreateHingeConstant(hinge);
+  }
+  else {
+    return hinge;
+  }
+}
+
+
+function CreateHingeVector(x, y, z) {
+  var hinge_vector = [x, y, z];
+  var hinge_vector_new = [0, 0, 0];
+  hinge_vector.forEach(function (hinge, index) {
+    hinge_vector_new[index] = CreateHinge(hinge);
+  });
+  return $V(hinge_vector_new[0], hinge_vector_new[1], hinge_vector_new[2]);
+}
+
+
+/**
+* Creates line hinge
+* @class
+* @constructor
+* @param	{Number}	no				Index of line hinge, can be undefined
+* @param	{String}	comment			Comment, can be undefined
+* @param	{Object}	params			line hinge parameters, can be undefined
+* @return	{Object}	Created line hinge
+*/
 function LineHinge(no,
```

Comment:
Add argument name.
LineHinge.prototype. = function(table_id, line, surface) {
Add argument AssignTo as two dimensional array - [[table_id1, line1, surface1],[table_id2, line2, surface2]] -> use function AssignTo in the constructor. USer get object and then modify conditions